_non-cosmos/solana/assetlist.json

```diff
@@ -229,6 +229,301 @@
+      "type_asset": "erc20",
+      "address": "edge86g9cVz87xcpKpy3J77vbp4wYd9idEV562CCntt",
+      "denom_units": [
+        {
+          "denom": "edge86g9cVz87xcpKpy3J77vbp4wYd9idEV562CCntt",
+          "exponent": 0
+        },
+        {
+          "denom": "edgesol",
+          "exponent": 9
+        }
+      ],
+      "base": "edge86g9cVz87xcpKpy3J77vbp4wYd9idEV562CCntt",
+      "name": "Edgevana Staked SOL",
+      "display": "edgesol",
+      "symbol": "edgeSOL",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/solana/images/edgesol.png"
+      },
+      "coingecko_id": "edgevana-staked-sol",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/solana/images/edgesol.png"
+        }
+      ],
+      "socials": {
+        "website": "https://edgevana.com/",
+        "twitter": "https://twitter.com/edgevana"
+      }
+    },
+    {
+      "description": "A liquid staked token that represents SOL staked to Helius's validator.",
+      "extended_description": "hSOL is a liquid representiation of staked SOL tokens to Helius, minted via Sanctum.",
+      "type_asset": "erc20",
+      "address": "he1iusmfkpAdwvxLNGV8Y1iSbj4rUy6yMhEA3fotn9A",
+      "denom_units": [
+        {
+          "denom": "he1iusmfkpAdwvxLNGV8Y1iSbj4rUy6yMhEA3fotn9A",
+          "exponent": 0
+        },
+        {
+          "denom": "hsol",
+          "exponent": 9
+        }
+      ],
+      "base": "he1iusmfkpAdwvxLNGV8Y1iSbj4rUy6yMhEA3fotn9A",
+      "name": "Helius Staked SOL",
+      "display": "hsol",
+      "symbol": "hSOL",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/solana/images/hsol.png"
+      },
+      "coingecko_id": "helius-staked-sol",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/solana/images/hsol.png"
+        }
+      ],
+      "socials": {
+        "website": "https://helius.dev/",
+        "twitter": "https://twitter.com/heliuslabs"
+      }
+    },
+    {
+      "description": "A liquid staked token that represents SOL staked to Jupiter's validator.",
+      "extended_description": "JupSOL is a liquid staking token that represents SOL staked to the Jupiter validator, via Sanctum.",
+      "type_asset": "erc20",
+      "address": "jupSoLaHXQiZZTSfEWMTRRgpnyFm8f6sZdosWBjx93v",
+      "denom_units": [
+        {
+          "denom": "jupSoLaHXQiZZTSfEWMTRRgpnyFm8f6sZdosWBjx93v",
+          "exponent": 0
+        },
+        {
+          "denom": "jupsol",
+          "exponent": 9
+        }
+      ],
+      "base": "jupSoLaHXQiZZTSfEWMTRRgpnyFm8f6sZdosWBjx93v",
+      "name": "Jupiter Staked SOL",
+      "display": "jupsol",
+      "symbol": "jupSOL",
```

Comment:
for the Solana LSTs, please define a trace to SOL as type: "liquid-staking"

Reply:
added